Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/config.json

```diff
@@ -0,0 +1,157 @@
+                "entry": "becknbase.html",
+                "root": ""
+            }
+        },
+        "in.juspay.upiintent": {
+            "default": {
+                "entry": "base.html",
+                "root": "payments/in.juspay.upiintent/"
+            }
+        }
+    },
+    "live": {
+        "assets": {
+            "in.juspay.ec": {
+                "config": "https://assets.juspay.in/juspay/payments/in.juspay.ec/release/v1-config.zip"
+            },
+            "in.juspay.godel": {
+                "acs_js_source": "https://assets.juspay.in/juspay/payments/in.juspay.godel/release/1.0rc2/v1-acs.zip",
+                "boot_loader_js_source": "https://assets.juspay.in/hyper/bundles/release/in.juspay.godel/android/1.0rc2/common/stable/v1-boot_loader.zip",
+                "config": "https://d3e0hckk6jr53z.cloudfront.net/godel/v1-config.zip"
+            },
+            "in.juspay.godel.placeholder": {
+                "acs_js_source": "https://assets.juspay.in/juspay/payments/in.juspay.godel/release/1.0rc2/v1-acs.zip",
+                "boot_loader_js_source": "https://assets.juspay.in/hyper/bundles/release/in.juspay.godel/android/1.0rc2/common/stable/v1-boot_loader.zip",
+                "config": "https://d3e0hckk6jr53z.cloudfront.net/godel/v1-config.zip"
+            },
+            "in.juspay.hyperos": {
+                "config": "https://assets.juspay.in/juspay/payments/2.0/release/v1-config.zip",
+                "manifest": "https://assets.juspay.in/hyper/bundles/in.juspay.merchants/jatrisaathidriver/android/release/manifest.json",
+                "sdk_config": "https://assets.juspay.in/juspay/atlas/in.juspay.jatrisaathidriver/release/common/sdk_config.json"
+            },
+            "in.juspay.hyperos.placeholder": {
+                "tracker": "https://assets.juspay.in/hyper/bundles/app/tracker/2.0.34/v1-tracker.zip"
+            },
+            "in.juspay.hyperpay": {
+                "configuration": "https://assets.juspay.in/hyper/bundles/in.juspay.merchants/juspay/configuration/8.2/v1-configuration.zip"
+            },
+            "in.juspay.upiintent": {
+                "config": "https://assets.juspay.in/juspay/payments/in.juspay.upiintent/release/v1-config.zip"
+            }
+        },
+        "id": "48215399535214850",
+        "package": {
+            "in.juspay.ec": "https://assets.juspay.in/juspay/payments/in.juspay.ec/release/2.0rc1/v1-index_bundle.zip",
+            "in.juspay.godel": "https://assets.juspay.in/juspay/payments/in.juspay.godel/release/2.0rc1/v1-index_bundle.zip",
+            "in.juspay.godel.placeholder": "https://assets.juspay.in/juspay/payments/in.juspay.godel/release/2.0rc1/v1-index_bundle.zip",
+            "in.juspay.hyperpay": "https://assets.juspay.in/juspay/payments/in.juspay.hyperpay/release/2.0rc1/v1-index_bundle.zip",
+            "in.juspay.jatrisaathidriver": "https://assets.juspay.in/hyper/bundles/app/in.juspay.jatrisaathidriver/1.1.0/android/v1-index_bundle.zip",
```

Comment:
@Vignesh-772 For jatrisaathidriver bundle push is setted up?

Reply:
not yet